### PR TITLE
Added a new Question

### PR DESCRIPTION
**Pull request by gauravshukla2203**
I am contributing here a new question in c++.md. If you are merging this PR , then please give it the tag of Hacktoberfest-accepted, as I have contributed this under Hacktoberfest 2022

### CODE REVIEW

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
merge conflicts

File: c++/c++quiz.md [hunk 1]
```diff
@@ -1405,5 +1405,11 @@ return 0;
 
 - [x] cout << "Hello World";
 - [ ] System.out.println("Hello World");
-- [ ] print("Hello World");
+- [ ] print("Hello World");
```
Comment (auchynnikau) [inline c++/c++quiz.md:1408]:
```suggestion
- [ ] print("Hello World");
```

**Comment by Ebazhanov** [comment]
@gauravshukla2203 ping

**Comment by Ebazhanov** [comment]
@gauravshukla2203 ping

**Comment by Ebazhanov** [comment]
@gauravshukla2203 your changes was merged via https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/pull/5084

**Pull request closed without merging**